js/controllers/window.js

```diff
@@ -28,8 +28,8 @@ function WindowController(editor, settings, analytics, tabs) {
   }
 }
 
-/** Initializes UI interaction animatins. */
-WindowController.prototype.initAnimations_ = function() {
+/** Initializes UI elements and animations. */
+WindowController.prototype.initUI_ = function() {
```

Comment:
I feel like the intent of the _ is to make this a private method. Can you please add the @private annotation to this patch please? :)

Reply:
Ack.